governance/program/src/state/realm_config.rs

```diff
@@ -0,0 +1,108 @@
+//! RealmConfig account
+
+use solana_program::{
+    account_info::AccountInfo, program_error::ProgramError, program_pack::IsInitialized,
+    pubkey::Pubkey,
+};
+
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+
+use crate::{
+    error::GovernanceError,
+    state::enums::GovernanceAccountType,
+    tools::account::{get_account_data, AccountMaxSize},
+};
+
+/// RealmConfig account
+/// The account is an optional extension to RealmConfig stored on Realm account
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub struct RealmConfigAccount {
```

Comment:
nit: it's not totally clear why this needs to be in a separate account.

Reply:
It only exist because I wanted to make it backward compatible and couldn't resize existing `Realm` accounts. 
Ideally we shouldn't need this and it could be added to `RealmConfig` stored on the Realm account but we need support to resize accounts on mainnet runtime first. 
There is also some small benefit for keeping it separate. If you don't use the advanced realm features you don't have to pay the extra rent, but I'm not sure if that alone would justify it.